src/js/helpers/cleanData.js

```diff
@@ -50,14 +50,21 @@ export const cleanLinks = (links, pathPrefix) => {
   if (!links || !links.edges) return null;
 
   return links.edges.map(({ node: link }) => {
-    const { title, text, slug, ...rest } = link;
-    return {
-      slug: slug,
-      url: `${pathPrefix || ''}/${slug}`,
-      text: title || text,
-      ...rest,
-    };
+    link.slug = link.slug || link.sortOrder;
+    link.url = `${pathPrefix || ''}/${link.slug}`;
+    link.text = link.text || link.title;
+    return link;
+  });
+};
+
+export const cleanProcesses = allProcesses => {
```

Comment:
now that we're thinking about Pearl's data retrieval, I'm wondering if these data cleaning utilities might have use there too...